src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView/configure-entries.jelly

```diff
@@ -6,42 +6,56 @@
-    </f:entry>
-
-    <f:entry title="Always allow manual trigger on pipeline steps" field="alwaysAllowManualTrigger">
-        <f:booleanRadio default="false"/>
-    </f:entry>
-
-    <f:entry title="Show pipeline project headers" field="showPipelineDefinitionHeader">
-        <f:booleanRadio true="Yes" false="No" default="false"/>
-    </f:entry>
-
-    <f:entry title="Show pipeline parameters in project headers" field="showPipelineParametersInHeaders">
-        <f:booleanRadio default="false"/>
-    </f:entry>
-
-    <f:entry title="Show pipeline parameters in revision box" field="showPipelineParameters">
-        <f:booleanRadio default="false"/>
-    </f:entry>
-
-    <f:entry title="Refresh frequency (in seconds)" field="refreshFrequency">
-        <f:textbox clazz="number"/>
-    </f:entry>
-
-    <f:entry title="URL for custom CSS files" field="cssUrl">
-        <f:textbox />
-    </f:entry>
-
-    <f:entry title="Console Output Link Style" field="consoleOutputLinkStyle">
-        <f:select />
-    </f:entry>
-
+    <f:section title="Pipeline Flow">
+        <f:dropdownDescriptorSelector field="gridBuilder" title="${%Layout}" />
+    </f:section>
+
+    <f:section title="Trigger Options">
+        <f:entry title="Restrict triggers to most recent successful builds" field="triggerOnlyLatestJob">
+            <f:booleanRadio default="true"/>
+        </f:entry>
+
+        <f:entry title="Always allow manual trigger on pipeline steps" field="alwaysAllowManualTrigger">
+            <f:booleanRadio default="false"/>
+        </f:entry>
+    </f:section>
+
+    <f:section title="Display Options">
+
+        <f:entry title="No Of Displayed Builds" field="noOfDisplayedBuilds">
+            <f:select />
+        </f:entry>
+        <!--
+        <f:entry title="Show pipeline project headers" field="showPipelineDefinitionHeader">
+            <f:booleanRadio true="Yes" false="No" default="false"/>
+        </f:entry>
+        -->
+
+        <f:dropdownDescriptorSelector field="rowHeaders" title="${%Row Headers}"
+                                      descriptors="${descriptor.getRowHeaderDescriptors()}"/>
+
+
+        <f:dropdownDescriptorSelector field="columnHeaders" title="${%Column Headers}"
+                                      descriptors="${descriptor.getColumnHeaderDescriptors()}"/>
+
+        <f:entry title="Refresh frequency (in seconds)" field="refreshFrequency">
+            <f:textbox clazz="number"/>
+        </f:entry>
+
+        <f:entry title="URL for custom CSS files" field="cssUrl">
+            <f:textbox />
+        </f:entry>
+
+        <f:entry title="Console Output Link Style" field="consoleOutputLinkStyle">
+            <f:select />
+        </f:entry>
+    </f:section>
+
+    <script type="text/javascript">
+        // dropdownDescriptorSelectors will fill items up but don't update the layout, so need
+        // to do that explicitly
+        setTimeout(function(){
+            layoutUpdateCallback.call();
```

Comment:
This was annoying AF - without this, the 'save / ok' button will overlap on the fields. If there's a better way to do it I'm open to it.